src/testkit/scala/tools/testkit/Unsymly.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Scala (https://www.scala-lang.org)
+ *
+ * Copyright EPFL and Lightbend, Inc.
+ *
+ * Licensed under Apache License 2.0
+ * (http://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package scala.tools.testkit
+
+import language.experimental.macros
+
+/** Facility for brief Symbol syntax.
+ *
+ *  `def x = Unsymly.mksym` would have liked `val x, y = mksym`
+ *
+ *  "Unseemly".
+ */
+object Unsymly {
+  def mksym: Symbol = macro SymlyImpl.impl
```

Comment:
Is that really necessary? There are two tests using this, with a grand total of ~15 symbol creations. Can't we just spell out the constructors instead of relying on macros for this?

Reply:
Yeah, it was more of a lark. But thanks for noticing!

Similarly, I'd skip the specialized `->` support for just warning suppression.